src/QsCompiler/Tests.Compiler/ExecutionTests.fs

```diff
@@ -236,7 +236,7 @@ type ExecutionTests(output: ITestOutputHelper) =
         assertEqual expected out
 
 
-    [<Fact>]
+    [<Fact(Skip = "Temporary failure.")>]
     member this.``QIR memory management``() =
```

Comment:
Can this skip be removed too?

Reply:
Woops, I removed that locally and forgot to push that. It is removed now.